### PR TITLE
Delete Fortran :D

### PR DESCRIPTION
**Pull request by AnthonyLim23**
This PR is removing the old code (Fortran and the direct translation). It also cleans up the code base, removing unused files and renames/restructures the code to suit the new code.

The "help" method of Python also works with the new package
```
>>> help("quickBayes")
Help on package quickBayes:

NAME
    quickBayes

PACKAGE CONTENTS
    fit_engines (package)
    fit_functions (package)
    log_likelihood
    utils (package)
    workflows (package)
```

To test:

see that the release pipieline is working: https://pypi.org/project/quickBayes/1.0.0b3/#history (windows and linux for 1.0.0b3) from this commit https://github.com/mantidproject/quickBayes/pull/57/commits/06e58808b708a8a628ba153f06ac981d48cf682b

Review the code and see that the Fortran is gone

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
my computer:

version      muon         qldata        qse       quest
c 0.3           0.087          0.349        4.99       72.3

**Comment by AnthonyLim23** [comment]
windows build server

```
version              muon                   qldata                qse                 quest
c 0.3                  0.087                    0.349                 4.99                  72.3
```

**Comment by AnthonyLim23** [comment]
ubuntu build server

```
version              muon                   qldata                qse                 quest
c 0.3                  0.087                    0.349                 4.99                  72.3
```

File: setup.py [hunk 1]
```diff
@@ -1,77 +1,36 @@
...
     name=PACKAGE_NAME,
+    requires=['numpy'],
     install_requires=['numpy>=1.12', 'scipy', 'gofit'],
-    packages=[PACKAGE_NAME],
+    packages=find_packages(where='src'),
     description='A Bayesian fitting package used for '
-                'fitting quasi-elastic neutron scattering data.',
-    long_description='This package wraps fortran Bayesian '
-                     'fitting libraries using f2py. '
+                'model selection and grid searchs of fits '
```
Comment (SilkeSchomann) [inline setup.py:19]:
```suggestion
                'model selection and grid searches of fits '
```

File: tools/setup_helper.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    return Extension(name=fq_name,
+                     sources=sources,
+                     include_dirs=[numpy.get_include()])
+
+
+def source_paths(dirname: str, filenames: Sequence[str]) -> Sequence[str]:
+    """
+    :param dirname: A relative path to the list of source files
+    :return: A list of relative paths to the given sources in the directory
+    """
```
Comment (SilkeSchomann) [inline tools/setup_helper.py:48]: param filenames is missing in docstrings

**Review by SilkeSchomann — CHANGES_REQUESTED** [review]
It is nice to see the Fortran code go! I've only found two absolutely minor things.

File: tools/setup_helper.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    :return: An Extension class to be built
+    """
+    return Extension(name=fq_name,
+                     sources=sources,
+                     include_dirs=[numpy.get_include()])
+
+
+def source_paths(dirname: str, filenames: Sequence[str]) -> Sequence[str]:
+    """
+    :param dirname: A relative path to the list of source files
```
Comment (AnthonyLim23) [inline tools/setup_helper.py:46]:
```suggestion
    :param dirname: A relative path to the list of source files
    :param filenames: the names of the source files
```

**Review by SilkeSchomann — APPROVED** [review]

**Comment by AnthonyLim23** [comment]
[test two](https://trello.com/c/Lf6KD7ya/4-test-two)

**Pull request merged**